docs/css/services.css

```diff
@@ -0,0 +1,115 @@
+/* Don't forget that some styles will be pulled from index.css. For instance, all helper tags should stay on index.css */
+
+/* Hero */
+.container .short-hero {
+	height: auto;
+	flex-flow: column nowrap;
+	padding: 20px 0px 0px 0px;
+	top: 50px;
+}
+
+.container .short-hero img {
+	margin-bottom: 20px;
+	width: 100%;
+	height: auto;
+}
+
+.container .short-hero .hero-content {
+	width: 80%;
+	text-align: left;
+}
+
+
+/* End Hero */
+/* Divider Line */
+.divider-line {
+	border-top: 2px solid black;
+	margin: 20px 0;
+}
+/* End Divider Line */
+/* Main Content */
+.container .main-content {
+	display: flex;
+	flex-flow: row wrap;
+	justify-content: center;;
+	width: 100%;
+}
+
+.container .main-content card {
+	width: 400px;
+	height: 250px;
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	box-sizing: border-box;
+	padding: 15px;
+	background-color: white;
+	box-shadow: 1px 1px 5px lightgray;
+	margin: 15px;
+	transition: height
+}
+
+.container .main-content card p {
+	width: 350px;
+}
+
+.container .main-content card:hover {
+	width: 410px;
+	height: 260px;
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	box-sizing: border-box;
+	padding: 25px;
+	background-color: white;
+	box-shadow: 1px 1px 30px lightgray;
+	margin: 15px;
+	transition: 1s;
+	transition-delay: 500ms;
+}
+
+.container .main-content card button {
+	width: 150px;
+	height: 30px;
+	border: 1px solid black;
+	background-color: transparent;
+	color: black;
+	border-radius: 5px;
+}
+
+.container .main-content card button:hover {
+	width: 150px;
+	height: 30px;
+	background-color: black;
+	color: white;
+	border-radius: 5px;
+	transition: 500ms;
+	transition-timing-function: ease-in-out;
+	cursor: pointer;
+}
+/* End Main Content */
+/* Final Section */
+.final-section {
+	display: flex;
+	flex-flow: row wrap;
+	justify-content: center;
+	height: 300px;
+}
```

Comment:
By adding an exact height here, it ends up messing with the footer placement on smaller screen widths.

![image](https://user-images.githubusercontent.com/8473052/57053689-bb33c000-6c5d-11e9-8278-9fe8e350f67a.png)

To get around this you'd need to either remove the height or add in a media query.